packages/lesswrong/lib/collections/jargonTerms/collection.ts

```diff
@@ -0,0 +1,40 @@
+import schema from './schema';
+import { createCollection } from '../../vulcan-lib';
+import { addUniversalFields, getDefaultResolvers } from '../../collectionUtils'
+import { getDefaultMutations, MutationOptions } from '../../vulcan-core/default_mutations';
+import { makeEditable } from "../../editor/make_editable";
+import { userCanDo, userOwns } from '@/lib/vulcan-users/permissions';
+import { Posts } from '../posts';
+
+const options: MutationOptions<DbJargonTerm> = {  
+  editCheck: async (user: DbUser|null, document: DbJargonTerm|null) => {
+    if (!user || !document) return false;
+    const post = await Posts.findOne({_id: document?.postId})
+
+    if (!post) return false;
+    return userOwns(user, post);
```

Comment:
Probably this should be checking permissions more similar to the editCheck on posts, i.e. checking for coauthors too. (But probably not podcasters/group organizers.)